indico/modules/categories/controllers/management.py

```diff
@@ -110,7 +110,7 @@ def _process(self):
         logo_form = CategoryLogoForm(obj=self.category)
 
         if form.validate_on_submit():
-            update_category(self.category, form.data, skip={'meeting_theme', 'lecture_theme'})
+            update_category(self.category, form.data, skip={'meeting_theme', 'lecture_theme'}, _extra_log_fields={})
```

Comment:
No need to pass this here

Reply:
Without passing it there, that parameter didn't appear in the method interception.

I found a solution passing it that way, is there another better solution?